workspaceitems.md

```diff
@@ -121,6 +121,19 @@ In addition, it allows in future to change the 1:1 association between collectio
 
 It returns the workspaceitem created by the specified submitter
 
+## Get Single Workspace Item from Item UUID
+
+**/api/submission/workspaceitems/item/<:uuid>**
+
+https://dspace7.4science.cloud/server/api/submission/workspaceitems/item/cd67ce0e-7f9a-42fc-b8e7-c8bb83ef58ca
+The item uuid is mandatory
+
+There's always at most one workspace item per Item.
+
+It would respond with:
+* The workspace item if there's a match
+* 404 if the workspace item doesn't exist
```

Comment:
Same here, I think there are additional status codes possible. Namely 401 and 403.